python-sdk/tests/benchmark/Makefile

```diff
@@ -23,23 +23,27 @@ clean:
 	@rm -f ../../unittests.db
 	@rm -f ../../airflow.cfg
 	@rm -f ../../airflow.db
+	@rm -f ../../../unittests.cfg
+	@rm -f ../../../unittests.db
+	@rm -f ../../../airflow.cfg
+	@rm -f ../../../airflow.db
```

Comment:
This feels odd. I think having two copies of everything leaves more surface area for confusion